packages/vue-components/src/Pic.vue

```diff
@@ -8,10 +8,10 @@
       class="img-fluid rounded"
       @load.once="computeWidth"
     />
-    <div class="image-caption">
+    <span class="image-caption">
```

Comment:
remark: necessary breaking change following #1548's definitions. shouldn't be too much of an issue since its the main use case https://github.com/MarkBind/markbind/issues/869.